base/cvd/cuttlefish/host/libs/confui/BUILD.bazel

```diff
@@ -0,0 +1,105 @@
+load("//:build_variables.bzl", "COPTS")
+load("//tools/lint:linters.bzl", "clang_tidy_test")
+
+package(
+    default_visibility = ["//:android_cuttlefish"],
+)
+
+genrule(
+    name = "roboto_medium_font",
+    srcs = [
+        "Roboto-Medium.ttf",
+    ],
+    outs = [
+        "roboto_medium.cc",
+    ],
+    cmd = "./$(location generate_font_cc.sh) \"$<\" RobotoMedium >\"$@\"",
```

Comment:
When running these against the internal build system, we'll have to use `$(location)` to get the paths to `xxd` and `sed` which are used inside the shell script. However IIUC `$(location)` expansion only works inside the `cmd` itself and not within the shell script.

Would it be possible to use a loop comprehension or macro rather than a shell script?

https://stackoverflow.com/a/48511351

Reply:
After further discussion we agreed to keep it this way since it seems the internal build system also supports this way of calling the commands.